src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -433,6 +433,22 @@ private void UpdateMSiTableGrid(Database msidb, string tableName)
 					// NOTE: Deliberately not calling msidb.TableExists here as some System tables could not be read due to using it.
 					string query = string.Concat("SELECT * FROM `", tableName, "`");
 
+					bool hasSequence = false;

```

Comment:
@learn-more One idea here to improve this would be to instead add the sorting logic into the view layer. This would prevent opening and close, and re-opening that database here. I think you could make it as simple as adding a call like this below just before the `Status();` call `this.View.TableViewSortBy("Sequence");` Then implement `TableViewSortBy` on `IMainFormView`/`MainForm` and if the specified column name doesn't exist just ignore it. The DataGrid can be sorted with [DataGridView.Sort Method](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.datagridview.sort?view=netframework-4.0#System_Windows_Forms_DataGridView_Sort_System_Windows_Forms_DataGridViewColumn_System_ComponentModel_ListSortDirection_). 

I haven't tried this so it's all theory, but I think that would be better than baking the SQL, re-opening of the DB, etc. behavior into this later. If you have concerns or think it's too much to ask, lets discuss!

Reply:
@activescott I'll have a go at that, will let you know how it works out.
If it works out fine, I'll force-push to this PR so that there is a clean commit history.

In my other PR you mentioned 'conventional commit syntax in the commit message.'
Is that something that I should do in my commits?

Reply:
> In my other PR you mentioned 'conventional commit syntax in the commit message.'
> Is that something that I should do in my commits?

It is ideal, but not required since I can squash-merge a PR and clean up history that way so not a /huge/ deal but all of this will be merged as a single commit and a single feature (e.g. "feat: usability improvements", albeit with a description). I think even more valuable than worrying about the commit history is to submit smaller PRs - like one for each feature/bug. I really like collaborating with others on this type of thing though, so I'll work through it with you in any case. Thanks again!!

Reply:
> > In my other PR you mentioned 'conventional commit syntax in the commit message.'
> > Is that something that I should do in my commits?
> 
> It is ideal, but not required since I can squash-merge a PR and clean up history that way so not a /huge/ deal but all of this will be merged as a single commit and a single feature (e.g. "feat: usability improvements", albeit with a description). I think even more valuable than worrying about the commit history is to submit smaller PRs - like one for each feature/bug. I really like collaborating with others on this type of thing though, so I'll work through it with you in any case. Thanks again!!

Due to the size of the individual issues I dismissed that idea, but if you prefer smaller single feature PR's I will try to keep that in mind 👍